### PR TITLE
Enlarge diamond shield

### PR DESCRIPTION
**Pull request by claysmalley**
The current diamond shield for Michigan and North Carolina state highways has teeny tiny text for 3-digit routes. 1- and 2-digit routes are legible, though have smaller text than the average shield. This PR attempts to make font size consistent with other shields. The design introduced here has a 24px width and height, though it is based on a 19px square rotated 45 degrees, so it should be visually similar to 20px rectangular shields.

200% zoom (retina)
----
before:
Screenshot from 2022-05-04 09-58-54

after:
Screenshot from 2022-05-04 09-57-33

100% zoom
----
before:
Screenshot from 2022-05-04 09-58-35

after:
Screenshot from 2022-05-04 09-58-07

### CODE REVIEW

File: style/js/shield_defs.js [hunk 1]
```diff
@@ -68,13 +68,14 @@ export function loadShields(shieldImages) {
...
+    backgroundImage: shieldImages.shield40_diamond_white,
+    textLayoutConstraint: ShieldText.ellipseTextConstraint,
     textColor: "black",
     padding: {
-      left: 5,
-      right: 5,
-      top: 5,
-      bottom: 5,
+      left: 2,
+      right: 2,
```
[the remark this answers is not in the document]
Comment (claysmalley) [reply style/js/shield_defs.js]: Tried out 2.5px spacing on left and right, and I think it looks better.

Screenshot from 2022-05-04 17-20-01
Screenshot from 2022-05-04 17-20-24

**Review by zekefarwell — APPROVED** [review]
Thanks!  This makes the visual weight of the diamond much more balanced with other shields.  A definite improvement.

**Comment by ZeLonewolf** [comment]
👍 screen shots look good to me.  Diamonds are a stupid shape for highway shields and Michigan and North Carolina DOTs should be ashamed of themselves.

**Comment by zekefarwell** [comment]
>  Diamonds are a stupid shape for highway shields and Michigan and North Carolina DOTs should be ashamed of themselves.

I'm in North Carolina now and can confirm.  The numbers on the diamond shields are small and harder than necessary to read at a distance.

**Pull request merged**